store/store.go

```diff
@@ -561,6 +575,10 @@ func (s *Store) newRequest(reqOptions *requestOptions, user *auth.UserState) (*h
 	req.Header.Set("X-Ubuntu-Series", s.series)
 	req.Header.Set("X-Ubuntu-Wire-Protocol", UbuntuCoreWireProtocol)
 
+	if os.Getenv("SNAPPY_DELTA_DOWNLOAD_FORMATS") != "" {
```

Comment:
Yeah, it's actually not a bool, but rather the supported formats for delta downloads. ie. "xdelta", or "xdelta,bsdiff" etc.